### PR TITLE
Create vet clinic table

### PR DESCRIPTION
**Pull request by ibugithub**
## In this branch I did the following:

- -[ ]  Create vet_clinic database,
- -[ ] Create animals table,
- -[ ]  Create columns for the table,
- -[ ] Insert data into the columns,
- -[ ]  Queried specific data within the table.

#### Find all animals whose name ends with "mon":
image

#### List the name of all animals born between 2016 and 2019.
image

#### List the name of all animals that are neutered and have less than 3 escape attempts.
image

#### List the date of birth of all animals named either "Agumon" or "Pikachu".
image

#### List name and escape attempts of animals that weigh more than 10.5kg
image

#### Find all animals that are neutered.
image

#### Find all animals not named Gabumon.
image

#### Find all animals with a weight between 10.4kg and 17.3kg (including the animals with weights that equals precisely 10.4kg or 17.3kg)
image

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -3,3 +3,12 @@
 CREATE TABLE animals (
     name varchar(100)
 );
+
+CREATE TABLE animals ( 
+    id SERIAL PRIMARY KEY,
```
Comment (t-yanick) [inline schema.sql]: - [ ] As per project requirements, the `id` field should be in `int` data type.
image

- [OPTIONAL ] You could also make the id column not null and autoincremented.
`    id int NOT NULL IDENTITY(1,1)
`

File: schema.sql [hunk 2]
```diff
@@ -3,3 +3,12 @@
 CREATE TABLE animals (
     name varchar(100)
 );
+
+CREATE TABLE animals ( 
+    id SERIAL PRIMARY KEY, 
+    name VARCHAR,
```
Comment (t-yanick) [inline schema.sql]: - [ ] The name column being of VARCHAR data type should have a limit. Example `name varchar(255),`

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @ibugithub , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Great job on this milestone. All SQL commands are in the correct files
:heavy_check_mark: Query results screenshots added to PR description
:heavy_check_mark: Use of correct Gitflow

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @ibugithub  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Made changes requested by the previous reviewer :+1: 
- Successfully created the animals table :gear: 

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**